src/resources/js/tickets-tables.js

```diff
@@ -14,10 +14,24 @@
 			return;
 		}
 
-		$element.sortable({
+		var fixHelper = function( e, ui ) {
+			ui.children().each( function() {
+				$( this ).width( $( this ).width() );
```

Comment:
Set the width to the width?

Reply:
To avoid the table collapsing when dragging it around.